contrib/local.mk.example

```diff
@@ -28,8 +24,24 @@
 #
 # CMAKE_BUILD_TYPE := Debug
 
-# By default, all libraries are statically linked to nvim. Uncomment these
-# entries to use dynamic linking instead.
+# By default, nvim uses bundled versions of its required third-party
+# dependencies.
+# Uncomment these entries to instead use system-wide installations of
+# them.
+#
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_BUSTED=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_DEPS=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_JEMALLOC=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_LIBTERMKEY=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_LIBUV=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_LIBVTERM=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_LUAJIT=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_LUAROCKS=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_MSGPACK=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_UNIBILIUM=OFF
```

Comment:
This won't work correctly if you uncomment two of them. :-)

It's probably better to do as below, and use `+=`.  `:=` is an assignment (with all variables fully expanded at the time of the assignment).